src/python/dxpy/utils/describe.py

```diff
@@ -444,6 +444,8 @@ def print_project_desc(desc, verbose=False):
         print_json_field("Restricted", desc["restricted"])
     if 'containsPHI' in desc:
         print_json_field('Contains PHI', desc['containsPHI'])
+    if 'databaseUIViewOnly' in desc:
```

Comment:
If this flag is not going to be publicized, do we want to print it out even when its set to false?

Reply:
I agree, let's print it only when the flag is set to true. Thanks for the suggestion!